code/_onclick/item_attack.dm

```diff
@@ -41,6 +41,15 @@
 	return FALSE
 
 /obj/attackby(obj/item/I, mob/living/user, params)
+	var/turf/T = (get_turf(src))
```

Comment:
You still need to remove this line you chumbis.

Reply:
Im not at home and i cant webedit it. Ill do it when i get home 

As well as some shit with altclickon so holding alt to walk works